### DISCUSSION TITLE
Saturday to Monday - 655 kilometers in Slovenia

### DISCUSSION THREAD

**Discussion by bios4** [devemux86/cruiser#552] [Show and tell]
Since my actual plans were (once again) severely thwarted, out of pure defiance I got on the Road King last Saturday and drove south without a precise plan...

This ultimately turned into just over two wonderful days of motorcycling with 655 kilometers on small and tiny roads in wonderful Slovenia - most of them unknown to me. 😎

Altough I planned all routes with Kurviger (since I still haven't found the right settings for BRouter to produce reasonable itineraries), all of them were navigated exclusively with Cruiser. Without any problems...

Really good job @devemux86! 👍

<img src="https://github.com/devemux86/cruiser/assets/145460147/c8ca35ab-1da6-4dd5-a76e-87bda311ba5e" width="24%">
<img src="https://github.com/devemux86/cruiser/assets/145460147/c7129ff1-4207-4782-8e43-f6574ad67eff" width="24%">
<img src="https://github.com/devemux86/cruiser/assets/145460147/9bbc7829-3baa-4a9e-a307-dc661fcbfe8a" width="24%">
<img src="https://github.com/devemux86/cruiser/assets/145460147/959e13ac-00a8-480a-9996-cbaf1e2786d9" width="24%">

**Comment by devemux86** [comment]
@bios4 thanks, very beautiful photos!
